CONTRIBUTING.md

```diff
@@ -0,0 +1,132 @@
+* Module files naming conventions closely follow the title of the corresponding section. For example, a section in the Getting Started document with the title *Working with pipelines* is named **working_with_pipelines.adoc** in the *modules* folder. In some cases, this is not a direct match, so to confirm the name of the asciidoc file for a section, click the section title and the ID after the *#* in the URL is the name of the module file. For example, a URL such as https://docs.openshift.io/getting-started-guide.html#viewing_build_pipeline_oso has a slightly different title but the ID after the hash symbol (**viewing_build_pipeline_oso**) is the name of the file.
+
+## Conventions and style guidelines
+
+To get started quickly, we recommend reviewing the following conventions that are relevant for this documentation suite:
+
+**Writing conventions**
+
+* For instructions, state the location, then action and subject. For example:
+  * Incorrect: *Click OK in the XYZ tab to continue.*
+  * Correct: *In the XYZ tab, click OK to continue.*
+  * Use gerunds in titles when possible. For example:
+    * Incorrect: *1.1 Create a new project*
+    * Correct: *1.1 Creating a new project*
+* All titles should be in sentence case. Exceptions for words that are proper nouns. For example:
+  * Incorrect: *1.2 Configuring Your Che Workspace*
+  * Correct: *1.2 Configuring your Che workspace*
+* URLs must be bared in the text, not hidden. For example:
+  * Incorrect: *Navigate to the [Red Hat website](www.redhat.com).* 
+  * Correct: *Navigate to (www.redhat.com).*
+  
+**Markup conventions**  
+
+* TBD 
+  
+For comprehensive references for documentation conventions, see the following:
+
+  * The IBM Style Guide. A subset of this guide is available [here](https://www.ibm.com/developerworks/library/styleguidelines/).
+  * The [Red Hat Asciidoc Markup conventions](https://redhat-documentation.github.io/asciidoc-markup-conventions/).
+  <!-- * The [CCS Documentation Conventions][url here when available] UNCOMMENT THIS WHEN READY-->
+
+## Editing the documentation
+
+To edit the documentation:
+
+1. Ensure that an issue with the details of the problem are logged at https://github.com/fabric8io/fabric8-online-docs/issues for context and assign it to yourself.
+
+2. Clone the documentation repository and make the relevant changes in a branch named after the issue you are working on.
+
+3. Locally build and test the changes. For details on the instructions and tools to do this, see [Locally building the documentation](#locally-building-the-documentation).
+
+3. After the local builds are successful, commit your changes and create a PR for the fixes. In the assignee field, select one of the writers to do a quick review before merging your changes. You can tag Robert Kratky, Misha Husnain Ali, or Preeti Chandrashekar as reviewers.
```

Comment:
Number 3 is repeated, using '.' instead of actual numbers in asciidoc helps avoid this.

Reply:
Do we want folks to assign us for review? and add our names here? I have not seen this practice, normally, we as maintainers of the repo would be expected to review any PR sent to the repo.